apps/web/src/pages/quick-start/steps/Setup.tsx

```diff
@@ -72,12 +78,7 @@ export function Setup() {
         {
           template: {
             type: StepTypeEnum.IN_APP,
-            content:
-              'Welcome to Novu! Click on this notification to <b>visit the cloud admin panel</b> managing this message',
-            cta: {
-              type: ChannelCTATypeEnum.REDIRECT,
-              data: { url: `/templates/edit/${notificationTemplate?._id}` },
-            },
+            content: 'Welcome to Novu! <b>visit the cloud admin panel</b> managing this message',
```

Comment:
I removed the CTA because the snippets do not have the click handlers, @scopsy let me know what you think.